demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Introduction\n",
+        "\n",
+        "This tutorial demonstrates how simulations performed via Inductiva API can be used in an optimization task.\n",
```

Comment:
"Inductiva API can effortlessly use simulation to determine the performance of various parameters in the design space." 

"In a previous notebook, we have demonstrated the exploration with a low-level call of the simulator. Here, we explain how the same can be done for a specific scenario and how we can post-process the data to achieve a desirable optimal design."

"We make use of ...."